diff --git a/examples/gcoap/gcoap_cli.c b/examples/gcoap/gcoap_cli.c
index 0f551214c..9dab4d704 100644
--- a/examples/gcoap/gcoap_cli.c
+++ b/examples/gcoap/gcoap_cli.c
@@ -257,6 +257,8 @@ static size_t _send(uint8_t *buf, size_t len, char *addr_str, char *port_str)
         return 0;
     }
 
+    gcoap_add_qstring(buf, "hello", "world");
+
     bytes_sent = gcoap_req_send(buf, len, &remote, _resp_handler);
     if (bytes_sent > 0) {
         req_count++;
